scripts/release/build-experimental-typescript.js

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env node
+
+/* eslint-disable import/no-commonjs */
+const fs = require('fs');
+const path = require('path');
+const shell = require('shelljs');
+
+const packageJsonPath = path.resolve('package.json');
+const packageJson = JSON.parse(fs.readFileSync(packageJsonPath).toString());
+const { version: currentVersion } = packageJson;
+const newVersion = `${currentVersion}-experimental-typescript`;
```

Comment:
This will fail when we release twice non-stable because multiple versions will be named the same.

- Actual
  - `4.2.0-experimental-typescript`
  - `4.2.0-experimental-typescript`
- Expected
  - `4.2.0-experimental-typescript.0`
  - `4.2.0-experimental-typescript.1`

We can ignore this if we only release for stable releases, but then we should append the version with `.0`.

Reply:
This comment was not addressed @eunjae-lee.

Reply:
@francoischalifour I thought it's resolved, because I updated this PR to release only for stable releases.

> We can ignore this if we only release for stable releases

https://github.com/algolia/instantsearch.js/pull/4307/commits/1e51fea844e2ae6f074efe82a574b1916ec00ea1

Reply:
> [...], but then we should suffix the version with `.0`.

To be confirmed, but I think tags need to be versioned.

Reply:
Yes, the current syntax isn't valid semver

Reply:
ah.. I get what you mean.